### ISSUE TITLE
[Bug] Some Sound Effects Appear to Come from the Wrong Direction and/or Changes Pitch

### ISSUE THREAD

**Issue by SonicRay100** [Mr-Wiseguy/Zelda64Recomp#263]
**Description**
Some sound effects appear to come from the wrong direction and remains at maximum volume even when the player is far away from the source. Additionally, the pitch of the sound may change based on the direction the camera is facing.

**To Reproduce**
1. Go to one of the moon child hide and seek rooms.
2. Move away from the blue teleporter.
3. Go into first person and pan the camera around.

**Expected behavior**
1. Sound effects should only come from the source.
2. Sound effect volume should disappear when far enough away.
3. Sound effect should not have pitch variations when moving the camera around.

**Desktop:**
- OS: Windows 10 Pro, 22H2
- OS Build: 19045.3930
- CPU: Intel(R) Core(TM) i7-6700K CPU @ 4.00GHz
- GPU: MSI GTX 1080 Gaming X 8GB
- RAM: 16 GB

**Additional context**
The moon child hide and seek rooms are the only ones that I know for sure but I have noticed this occur in other places.

I made a video to that demonstrates the bug:

https://www.youtube.com/watch?v=IN7KuXe2qNk

**Comment by Reonu** [comment]
Hello, thanks for the report. 

This is vanilla and accurate behavior. Here I've attached footage recorded on my own N64 through a capture card:

https://github.com/Zelda64Recomp/Zelda64Recomp/assets/15913880/0783df32-f50c-458c-b09c-bf9b7aa603c7

Fixing vanilla issues is outside of the scope of this project and better suited for a mod when mod support releases. That being said, should anyone wish to make a PR to improve this aspect of the game, it might get approved if isn't too invasive.

Closing since this is not an issue with the project.
